ro-crate-metadata.json

```diff
@@ -121,14 +127,33 @@
         },
         {
             "@id": "main.nf",
-            "@type": ["File", "SoftwareSourceCode", "ComputationalWorkflow"],
+            "@type": [
+                "File",
+                "SoftwareSourceCode",
+                "ComputationalWorkflow"
+            ],
             "creator": [
+                {
+                    "@id": "#68351153+CarsonJM@users.noreply.github.com"
+                },
+                {
+                    "@id": "#42973691+d4straub@users.noreply.github.com"
+                },
                 {
                     "@id": "https://orcid.org/0000-0001-9807-1082"
+                },
+                {
+                    "@id": "https://orcid.org/0000-0003-0603-7907"
+                },
+                {
+                    "@id": "#jfy133@gmail.com"
+                },
+                {
+                    "@id": "#jfy133@gmail.com"
```

Comment:
?

Reply:
I dunno, that gets geneated autmoatically!?

Reply:
Ah because it'I've added that info in the manifest